src/mobile/multichoice/test/items.test.ts

```diff
@@ -50,6 +56,35 @@ describe("MobileMultiChoice", () => {
       expect(container.items).to.be.equal(initItems);
     });
 
+    it("exists on element and set item label the same as value when initializing with props option without label", async () => {
+      const container = new MobileMultiChoice({ items: initItemsWithoutLabel });
+
+      const el = await fixture(container);
+      const itemsEl = el.getElementsByTagName("option");
+      expect(itemsEl.length).to.be.equal(3);
+
+      for (let i = 0; i < itemsEl.length; i++) {
```

Comment:
Please don't use for in unit test
ref: https://sharedoc.atlassian.net/wiki/spaces/SC/pages/2430697566/EN+KUC+v1.x.x+-+Rules+to+Better+Unit+Tests#Don't-use-loops-and%2For-conditionals